CHANGES.md

```diff
@@ -1,3 +1,37 @@
+## edge-20.4.3
+
+This edge release adds functionality to the CLI to output more detail and
+includes changes which support the multi-cluster functionality. Also, the helm
+support has been expanded to make installation more configurable. Finally, the
+HA reliability is improved by ensuring that control plane pods are restarted
+with a rolling strategy
+
+* CLI
+  * Added output to the `linkerd check --proxy` command to list all data
+    plane pods which are not up-to-date rather than just printing the first
+    one it encounters
+  * Added a `--proxy` flag to the `linkerd version` command which lists all
+    proxy versions running in the cluster and the number of pods running
+    each version
+  * Added MeshedPodCount field to TrafficSplit resource rows
```

Comment:
A user-facing description would rather be something like "Lifted requirement of using `--unmeshed` for `linkerd stat` when querying TrafficSplit resources"

Reply:
👍